frontend/src/metabase/reference/components/EditHeader.css

```diff
@@ -18,7 +14,7 @@
 
 :local(.editHeaderButton) {
   border: none;
-  color: var(--edit-header-color);
+  color: var(--color-brand);
```

Comment:
This doesn't have anything to do with this PR as I have confirmed this also happens on `master` but I figure I should inform you that this component CSS is broken since you're editing it.

![Screenshot 2023-08-07 at 7 39 09 PM](https://github.com/metabase/metabase/assets/1937582/c400c495-578f-40f4-be8e-65c459011350)

You won't see `You are editing this page` unless you highlight the text. I mean if you see it without highlighting the text then you have better eyes than everyone else in the world for sure.
![Screenshot 2023-08-07 at 7 39 04 PM](https://github.com/metabase/metabase/assets/1937582/d0558fb8-59c8-4944-bed7-9597275ccea9)